client/src/components/uploads/uploadCodeSystem/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, ButtonGroup, InputGroup } from '@blueprintjs/core';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import axios from 'axios';
+import { FHIR_API_URL } from '../../../constants';
+
+import { IReduxStore } from 'types';
+
+import { validator } from 'components/uploads/validate';
+import fhirSchema from 'components/uploads/uploadCodeSystem/CodeSystem.schema.json';
+
+import './style.scss';
+
+const UploadCodeSystem = (): React.ReactElement => {
+  const toaster = useSelector((state: IReduxStore) => state.toaster);
+
+  const [newCodeSystem, setNewCodeSystem] = useState(undefined as any);
+  const [uploading, setUploading] = useState(undefined as any);
+  const [invalidFile, setInvalidFile] = useState(false);
+
+  const validate = validator(fhirSchema);
+
+  const validateCodeSystem = (codeSystem: any): boolean => {
```

Comment:
I don't think you need this function.. can't you use `validate` directly ? it returns a boolean

Reply:
That's right! I tried to keep the same structure as in uploadProfile but I don't do anything anymore in this function. I'll remove it